### PR TITLE
LPS-118086-Crypto

### PR DESCRIPTION
**Pull request by csierra**
/cc @arthurchan35 @martamedio

Hi Brian, 

here comes the first PR for the new hashing framework. We are sending the API module only. 

Bests. 

Carlos.

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hash-pepper-storage-select=Hash Pepper Storage OSGi Target Filter Selector
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

File: modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties (hunk 1 above)
Comment (csierra) [reply modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties:1]: looks like they are

**Comment by csierra** [comment]
ci:forward

**Comment by csierra** [comment]
failures unrelated... sent to https://github.com/brianchandotcom/liferay-portal/pull/96812

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:close

**Pull request closed without merging**